examples/nlp/text_classification/conf/text_classification_config.yaml

```diff
@@ -75,8 +73,8 @@ model:
 
     sched:
```

Comment:
Optimizers and scheduler config has been simplified. See https://github.com/NVIDIA/NeMo/blob/e97aad2cd33bddb6937b9553b7d5b07b21f5fe1f/examples/asr/conf/config.yaml#L139

Reply:
Done